src/base1/test-chan.js

```diff
@@ -337,31 +339,32 @@ QUnit.test("queue messages", function (assert) {
     });
 });
 
-QUnit.test("receive message", function (assert) {
+QUnit.skip("receive message", function (assert) {
```

Comment:
Why do you skip bunch of tests?

Reply:
Argh, crap! That was the one I was debugging for ages, and then I forgot to take out the skip again. Thanks for spotting! I'll most probably revert to the "port most tests" commit again, and leave that "receive message" sucker for later (the *only* test that throws a spanner into the works, and I just can't figure out for the life of me why the message isn't being sent).